### PR TITLE
Prep 0.2.0 release

### PR DESCRIPTION
**Pull request by alinabuzachis**
Depends-on: https://github.com/ansible/ansible-zuul-jobs/pull/1616

##### SUMMARY

Run add_docs, generate changelog

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by alinabuzachis** [comment]
recheck

**Comment by gravesm** [comment]
recheck

**Comment by gravesm** [comment]
recheck

**Review by gravesm — COMMENTED** [review]
I confess I didn't read all 20,000 lines of changes :sweat_smile:. Everything in `module_utils` has already gone through review. I picked a couple modules and looked at changes for the module and docs to make sure the generation process turned out ok. Overall, looks good. Spotted a couple small things.

**Review by jillr — COMMENTED** [review]
I also did not exhaustively read the generated docs but did catch a couple things.

File: plugins/modules/kms_replica_key.py [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+# See: https://github.com/ansible-collections/amazon_cloud_code_generator
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+module: kms_replica_key
+short_description: Creates andn manages a multi-Region replica key that is based on
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply plugins/modules/kms_replica_key.py]: Fix in the docs and in the generator repo https://github.com/ansible-collections/amazon_cloud_code_generator/pull/49/files

File: plugins/modules/cloudtrail_event_data_store.py [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+module: cloudtrail_event_data_store
+short_description: Creates and manages a new event data store
+description:
+- Creates and manages Creates a new event data store.a new event data store.
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply plugins/modules/cloudtrail_event_data_store.py]: Fixed in the docs and in the generator repo https://github.com/ansible-collections/amazon_cloud_code_generator/pull/49/files

File: changelogs/changelog.yaml [hunk 3]
```diff
@@ -1 +1,137 @@
...
+      name: s3_multi_region_access_point_policy
+      namespace: ''
+    - description: Create and manage Object Lambda Access Points used to access S3
+        buckets
+      name: s3_object_lambda_access_point
+      namespace: ''
+    - description: Specifies the Object Lambda Access Point resource policy document
+      name: s3_object_lambda_access_point_policy
+      namespace: ''
+    release_date: '2022-08-23'
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply changelogs/changelog.yaml]: Updated to April 11th.

File: docs/amazon.cloud.cloudtrail_event_data_store_module.rst [hunk 4]
```diff
@@ -0,0 +1,702 @@
...
+
+
+
+Requirements
+------------
+The below requirements are needed on the host that executes this module.
+
+- boto3 >= 1.17.0
+- botocore >= 1.20.0
+- python >= 3.6
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply docs/amazon.cloud.cloudtrail_event_data_store_module.rst]: Done!

**Comment by alinabuzachis** [comment]
recheck

File: plugins/modules/cloudtrail_event_data_store.py [hunk 5]
```diff
@@ -15,7 +15,7 @@
 module: cloudtrail_event_data_store
 short_description: Creates and manages a new event data store
 description:
-- Creates and manages Creates a new event data store.a new event data store.
+- Creates and manages Creates a new event data store.
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply plugins/modules/cloudtrail_event_data_store.py]: @jillr Done, thanks!

File: docs/docsite/rst/guide_amazon_cloud.rst [hunk 6]
```diff
@@ -1,124 +0,0 @@
-.. _ansible_collections.amazon.cloud.docsite.aws_intro:
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply docs/docsite/rst/guide_amazon_cloud.rst]: Missed this comment. No, it shouldn't. My bad, I deleted the docs folder to ensure that the docs have been re-generated and I forgot about docsite folder inside. Restored. Thank you.

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

File: plugins/modules/cloudtrail_event_data_store.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (gravesm) [reply plugins/modules/cloudtrail_event_data_store.py:18]: Maybe I'm still seeing on old cached version, but it doesn't look like the change in the generator collection is reflected here. What I'm seeing: `Creates and manages Creates a new event data store.`. This should read `Creates and manages a new event data store.`

[the remark this answers is not in the document]
Comment (alinabuzachis) [reply plugins/modules/cloudtrail_event_data_store.py]: Should be fixed now! Thanks.

**Review by gravesm — APPROVED** [review]

**Pull request merged**